### PR TITLE
[5.0] Guidedtour use dialog to show all tours, instead of bs modal

### PR DESCRIPTION
**Pull request by Fedik**
Pull Request for Issue # .

### Summary of Changes

Use Dialog to show "all guidedtours", instead of hardcoded BS modal.

BTW. It also can be changed to use AJAX for loading all tours, in future.

### Testing Instructions
Apply patch.
Click "Show all tours" 

### Actual result BEFORE applying this Pull Request
You get a popup with all tours

### Expected result AFTER applying this Pull Request
You get a popup with all tours, but better coded :)

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed
- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

Reference:
 - https://github.com/joomla/joomla-cms/pull/40150

### CODE REVIEW

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 11d3a6317b9ba710f6e6f615109d52b2f77c9ab1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41442">issues.joomla.org/tracker/joomla-cms/41442</a>.</sub>

File: administrator/modules/mod_guidedtours/tmpl/default.php [hunk 1]
```diff
@@ -23,7 +22,9 @@
 // Load the Bootstrap Dropdown
 $app->getDocument()
     ->getWebAssetManager()
-    ->useScript('bootstrap.dropdown');
+    ->useScript('bootstrap.dropdown')
+    ->useScript('joomla.dialog-autocreate')
+;
```
Comment (heelc29) [inline administrator/modules/mod_guidedtours/tmpl/default.php]:
```suggestion
    ->useScript('joomla.dialog-autocreate');
```

**Review by heelc29 — COMMENTED** [review]
is that intended?

**Comment by rachelwalraven** [comment]
I have tested this item :white_check_mark: successfully on 11d3a6317b9ba710f6e6f615109d52b2f77c9ab1<br /><br />Tested succesfully<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41442">issues.joomla.org/tracker/joomla-cms/41442</a>.</sub>

**Comment by heelc29** [comment]
I have tested this item :white_check_mark: successfully on b02b0df813c9053091cf132a0d5b78bf2002968c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41442">issues.joomla.org/tracker/joomla-cms/41442</a>.</sub>

**Comment by alikon** [comment]
@rachelwalraven can you please re-test cause meanwhile a commit has been done 
thanks in advance

**Comment by ChristineWk** [comment]
I have not tested this item.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41442">issues.joomla.org/tracker/joomla-cms/41442</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on b02b0df813c9053091cf132a0d5b78bf2002968c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41442">issues.joomla.org/tracker/joomla-cms/41442</a>.</sub>

**Comment by ChristineWk** [comment]
sorry, marked "not tested" done by error.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41442">issues.joomla.org/tracker/joomla-cms/41442</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41442">issues.joomla.org/tracker/joomla-cms/41442</a>.</sub>

**Comment by web54** [comment]
I have tested this item ✅ successfully
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41442">issues.joomla.org/tracker/joomla-cms/41442</a>.</sub>

**Comment by ceford** [comment]
My list of guided tours, and the dialog box, are showing string keys rather than values. This both with and without the patch applied. And all of the key="string" titles and descriptions are in separate files. How come?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41442">issues.joomla.org/tracker/joomla-cms/41442</a>.</sub>

**Comment by Fedik** [comment]
I think it is something broken in general, The translations is missing.
Seems one of recent changes broke it.

**Comment by HLeithner** [comment]
thx

**Pull request merged**